examples/play.rs

```diff
@@ -14,14 +15,18 @@ fn main() {
         graph.create_node(AudioNodeType::GainNode(options));
         assert_eq!(graph.current_time(), 0.);
         graph.resume();
+        // change frequency at 0.5s and 1s
+        // change gain at 0.75s and 1.25s
+        graph.message_node(0, AudioNodeMessage::SetAudioParamEvent(UserAutomationEvent::SetValueAtTime(110., 0.5)));
+        graph.message_node(0, AudioNodeMessage::SetAudioParamEvent(UserAutomationEvent::SetValueAtTime(220., 1.)));
```

Comment:
Yeah so the plan is to have an enum full of variants for each kind of param. I just didn't implement that here since it's trivial to do any time and we don't yet have anything with more than one param.